### ISSUE TITLE
Variable VM_CONTAINERREGISTRY behavior

### ISSUE THREAD

**Issue by RaSerge** [VictoriaMetrics/operator#654]
Hello!

I tried to run vm-operrator in private network without internet access.
i set operator.useCustomConfigReloader: false because of https://github.com/VictoriaMetrics/operator/issues/653
Then i tried three variants:

1. i set
```
VM_CONTAINERREGISTRY: myinternalregistry.example.com
VM_VMAGENTDEFAULT_CONFIGRELOADIMAGE: prometheus-operator/prometheus-config-reloader:v0.58.0
```
create vmagent cr and  see
agent image path:
```myinternalregistry.example.com/victoriametrics/vmagent:v1.89.1```
reloader image path:
```prometheus-operator/prometheus-config-reloader:v0.58.0```
->no registry path  in reloader image!

2. i set
```
VM_CONTAINERREGISTRY: myinternalregistry.example.com
VM_VMAGENTDEFAULT_CONFIGRELOADIMAGE: myinternalregistry.example.com/prometheus-operator/prometheus-config-reloader:v0.58.0
```
create vmagent cr and  see
agent image path:
```myinternalregistry.example.com/victoriametrics/vmagent:v1.89.1```
reloader image path:
```myinternalregistry.example.com/myinternalregistry.example.com/prometheus-operator/prometheus-config-reloader:v0.58.0```
->registry path and image in reloader image are concatenated!

3. i set only
```
VM_VMAGENTDEFAULT_CONFIGRELOADIMAGE: myinternalregistry.example.com/prometheus-operator/prometheus-config-reloader:v0.58.0
```
create vmagent cr and  see
agent image path:
```docker.io/victoriametrics/vmagent:v1.89.1```
reloader image path:
```myinternalregistry.example.com/prometheus-operator/prometheus-config-reloader:v0.58.0```
->looks good

IMO at least 1 looks like a bug (VM_CONTAINERREGISTRY not added to VM_VMAGENTDEFAULT_CONFIGRELOADIMAGE)

Thank you!
```
operator Chart version: 0.22.0
operator version: v0.34.0
```

**Comment by f41gh7** [comment]
Hello, thanks for reporting it's an issue with new feature - image is not correctly set for initContainer. It's fixed at the following commit https://github.com/VictoriaMetrics/operator/commit/65d8d0a6ec2614c69fe9456dbab4cb62ad0c9d04

 I expect patch release today.

**Comment by f41gh7** [comment]
An issue must be fixed at [v0.34.1 release](https://github.com/VictoriaMetrics/operator/releases/tag/v0.34.1)
